README.md

```diff
@@ -367,28 +367,40 @@ Registered object succesfully!
 
 <span class="notes">**Note:** Device name is the endpoint name you will need later on when [testing the application](https://github.com/ARMmbed/mbed-os-example-client#testing-the-application).</span>
 
-When you press the **SW2** button on your board you should see messages about the value changes:
+When you press the **BUTTON1** button on your board you should see messages about the value changes:
 
 ```
 handle_button_click, new value of counter is 1
 ```
+## Which button is BUTTON1 or BUTTON2?
 
+The abstract button definitions are not printed on the boards. The easiest way to find out which button is which is to grep the code. Example below finds the `BUTTON1` for K64F.
+
+```bash
+<shell: mbed-os-example-client/> cd mbed-os
+<shell: mbed-os-example-client/mbed-os>$ git grep BUTTON1 |grep -i K64F
+targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/TARGET_FRDM/PinNames.h:    BUTTON1 = SW2,
+```
```

Comment:
This one will not work straight out of the box, because the buttons were ENUMs... We need to figure out an alternative solution.